lib/tag-handler.js

```diff
@@ -40,7 +40,7 @@ module.exports.tagHandler = (command) => async ({ github, verbose, number }) =>
 
     if (tagVersion !== version) {
       console.error('Intended version bump does not match package.json version field')
-      process.exit(1)
+      process.exit(0)
```

Comment:
is this intended? if so then we should perhaps reflect the console message to be `console.log` instead of `console.error`

Reply:
it is intended. I'm not sure it's a good idea to error the whole workflow and have the repo with that misleading `X` due to no match json right? Just the log can be enough, maybe even using your logging function actually...

Reply:
then let's change the log above to `console.log()` should do